app/controllers/events_controller.rb

```diff
@@ -1,5 +1,21 @@
 class EventsController < ApplicationController
-  def create; end
+  def new
+    @friend_id         = params[:friend_id]
+    @friend_name       = params[:friend_name]
+    @gym_membership_id = params[:gym_membership_id]
+    @gym_name          = params[:gym_name]
+  end
+
+  def create
+    new_event = BackEndFacade.create_event(create_event_params)
+    if new_event.present?
+      flash[:alert] = 'Workout created! Now go get your swell on!'
+      redirect_to dashboard_path(current_user.id)
+    else
+      flash[:alert] = 'Error: Something went wrong!'
+      redirect_to URI(request.referer).path
```

Comment:
Gonna have to circle back to wrap my head on this syntax when more time

Reply:
This just sends the user back to the place they came from (in this case, the new event page).